src/cozmo/world.py

```diff
@@ -590,73 +590,207 @@ def send_available_objects(self):
+        try:
+            msg = await self.wait_for(_clad._MsgDefinedCustomObject, timeout=5)
+        except asyncio.TimeoutError as e:
+            logger.error("Failed (Timed Out) to define: %s", custom_object_archetype)
+            return None
+
+        msg = msg.msg  # get the internal message
+        if msg.success:
+            type_id = custom_object_archetype.object_type.id
+            self.custom_objects[type_id] = custom_object_archetype
+            logger.info("Defined: %s", custom_object_archetype)
+            return custom_object_archetype
+        else:
+            logger.error("Failed to define Custom Object %s", custom_object_archetype)
+            return None
+
+    async def define_custom_box(self, custom_object_type,
+                                marker_front, marker_back,
+                                marker_top, marker_bottom,
+                                marker_left, marker_right,
+                                depth_mm, width_mm, height_mm,
+                                marker_width_mm, marker_height_mm,
+                                is_unique=True):
         '''Defines a cuboid of custom size and binds it to a specific custom object type.
 
-        Warning: This function is currently experimental and has several known issues.
-        1) There seems to be an off by 10x issue related to how the vision reports the object's
-        size and position.
-        2) The ID returned for these objects is not consistent.
-        3) Poor performance and other issues have been seen in the App when using this.
-        We plan to expand and improve upon it in a future release before making
-        it fully public and documented.
+        The engine will now detect the markers associated with this object and send an
+        object_observed message when they are seen. The markers must be placed in the center
+        of their respective sides.
+
+        Args:
+            custom_object_type (:class:`cozmo.objects.CustomObjectTypes`): the
+                object type you are binding this custom object to
+            marker_front (:class:`cozmo.objects.CustomObjectMarkers`): the marker
+                affixed to the front of the object
+            marker_back (:class:`cozmo.objects.CustomObjectMarkers`): the marker
+                affixed to the back of the object
+            marker_top (:class:`cozmo.objects.CustomObjectMarkers`): the marker
+                affixed to the top of the object
+            marker_bottom (:class:`cozmo.objects.CustomObjectMarkers`): the marker
+                affixed to the bottom of the object
+            marker_left (:class:`cozmo.objects.CustomObjectMarkers`): the marker
+                affixed to the left of the object
+            marker_right (:class:`cozmo.objects.CustomObjectMarkers`): the marker
+                affixed to the right of the object
+            depth_mm (float): depth of the object (in millimeters) (X axis).
```

Comment:
In the depth_mm, width_mm and height_mm lines, I don't think we need a period after the "(* axis)." If we have a period after "(in millimeters)" it should uniform across the 3 lines.